src/hooks/signMessage/useSignMessage.ts

```diff
@@ -0,0 +1,262 @@
+import { useEffect, useState } from 'react';
+import { CANCELLED, ERROR_SIGNING } from 'constants/index';
```

Comment:
why not use SignedMessageStatusesEnum.cancelled and SignedMessageStatusesEnum.error ?

Reply:
Because we need to show exact error messages